build.rs

```diff
@@ -286,34 +243,79 @@ fn make_printable(collected: Vec<Record>) -> Vec<String> {
         .collect()
 }
 
-fn main() {
+fn main() -> std::io::Result<()> {
+    use std::io::{BufRead, BufReader};
+    use std::fs::File;
+
     let out_path = PathBuf::from(env::var("OUT_DIR").unwrap());
-    let prefix = out_path.join("lightning-prefix");
-    let libdir = prefix.join("lib");
-    let incdir = prefix.join("include");
+    let base = PathBuf::from("vendor/gnu-lightning");
+    let incdir = base.join("include");
+    let libdir = base.join("lib");
+
+    {
+        let input = BufReader::new(File::open(incdir.join("lightning.h.in"))?);
+        let mut output = File::create(out_path.join("lightning.h"))?;
+        for line in input.lines() {
+            let line = line?;
+            match line.as_str() {
+                "@MAYBE_INCLUDE_STDINT_H@" => writeln!(output, "#include <stdint.h>")?,
+                _ if !line.contains('@') => writeln!(output, "{}", line)?,
+                _ => panic!("Found unexpected automake token in lightning header"),
+            }
+        }
+    }
 
-    if !lightning_built(&prefix) {
-        build_lightning(prefix.to_str().unwrap()).unwrap();
+    let definitions = &[
+        // TODO make HAVE_FFSL dependent on target (it is in POSIX, but Windows
+        // might not have it)
+        ("HAVE_FFSL", None),
+        // TODO remove NDEBUG -- it works works around an invalid assertion
+        // (`assert(l != h)`) in _jit_new_node_qww
+        ("NDEBUG", None),
+    ];
+
+    let files = &[
+        libdir.join("jit_disasm.c"),
```

Comment:
Since `DISASSEMBLER` is not defined during compilation, disassembly remains inert. I have a separate branch where I am working on disassembly for #5.